content/doc/administrate/tcp-redirections.md

```diff
@@ -11,47 +11,98 @@ keywords:
 type: docs
 ---
 {{< callout type="warning" >}}
-TCP redirections are currently available free of charge as long as the feature is still considered in a BETA stage.
+TCP redirections are currently available free of charge as long as the feature is in BETA stage.
 Once the feature leaves the BETA stage, it won't be free anymore.
 {{< /callout >}}
 
 ## What is a TCP redirection?
 
-A regular application on Clever Cloud needs to listen on port 8080 and answer to HTTP traffic.
-Our reverse proxies then make your application available using HTTP(s) using your domain name.
+Every application hosted Clever Cloud must listen on port 8080 and answer to HTTP traffic.
+This is useful for both routing requests and monitoring purposes.
 
-Some applications might additionally want to use a binary protocol, with raw TCP interactions.
-For this scenario, your application will have to listen for TCP traffic on port 4040.
-Once this is done, all you need to do is add a TCP redirection to your application. A port will be
-provided to you, and your TCP listener will be available on your domain name using the provided port.
+However, some applications might additionnally require binary protocol, with raw TCP interactions.
+In that case, you can use a TCP redirection to route TCP traffic, through our reverse proxies, to your application.
+
+Every application can be configured to receive TCP traffic on the port 4040.
+We then assign a specific port to your application to expose it to TCP traffic. This port is chosen at random above 5000.
```

Comment:
```suggestion
Clever Cloud then assign a specific port to your application to expose it to TCP traffic. This port is chosen at random above 5000.
```

Replace "we" by "Clever Cloud

Reply:
It would become "Clever cloud then assigns".

Reply:
```suggestion
Clever Cloud then assigns a specific port to your application to expose it to TCP traffic. This port is chosen at random above 5000.
```